src/apiClient.cpp

```diff
@@ -114,7 +114,7 @@ std::string apiClient::sendPOSTRequest() {
     return "Error: " + errorToString(res.error());
 }
 
-std::string apiClient::errorToString(httplib::Error err) {
+std::string apiClient::errorToString(const httplib::Error& err) const {
```

Comment:
enum class httplib::Error is actually an int-sized data value, so `const&` for this is not necessarily optimal (although there's not a lot of difference anyway), but it might be better to allow for the caller to hold a `const&` and then pass it in -- that's the viral nature of `const`.

In short, C++ `const` is a whole lot of subtlety. After a long time of resisting it as a hassle I've come around to the idea that it's far better to use it then to avoid it.